tests/legacy-cli/e2e/utils/process.ts

```diff
@@ -10,11 +10,27 @@ const treeKill = require('tree-kill');
 interface ExecOptions {
   silent?: boolean;
   waitForMatch?: RegExp;
-  env?: { [varname: string]: string };
+  env?: { [varname: string]: string } | 'npm';
   stdin?: string;
   cwd?: string;
 }
 
+const NPM_CONFIG_RE = /^npm_config_/i;
+
+function extractNpmEnv() {
```

Comment:
This is a bit of a hack and I'd like to find a better way but wanted to push up what I think is a working version that keeps the `--global`.